### DISCUSSION TITLE
Is there any comprehensive documentation about all the features available?

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomlatools/joomlatools-pages-server#101] [Q&A] [answered]
Hi @johanjanssens,

Everything is in the question. Basically, will there be a more comprehensive version of the wiki in the near future or would we need to dig deeper in the code ourselves to figure out how it works? I don't mean to be offensive. It's just a question to clarify what to expect in near future.

**Comment by johanjanssens** [comment, marked as answer]
Trying to finish up a first release and will also work a bit more on docs. For now, if you go through the different PR's you will find quite a bit of info how everything works: https://github.com/joomlatools/joomlatools-server/pulls?q=is%3Apr+is%3Aclosed

**Reply by alexandreelise** [reply]
Hi johan and his team! Keep up the awesome work you are doing!. I found out a gem in the public project timeline it's how to generate a minica certificate using joomla console the command is 
``` bin/jtctl bash  then do a joomla cert:create [yoursitename].test ```  nice work folks! keep up. Will dig in the PRs now. Thanks Johan.

**Reply by johanjanssens** [reply]
Joomlatools Server is a fully working local development environment for Joomla. The whole team is using it daily for all our development. It includes everything you need and then a bit more. The PR's are pretty complete, just not had the time to put it into a wiki yet.

**Reply by alexandreelise** [reply]
when you say local development it's not ready for production yet?

**Reply by johanjanssens** [reply]
See: https://github.com/joomlatools/joomlatools-server#what-is-joomlatools-server 

Its a replacement for MAMP/XAMP etc, or a custom stack. Cloud Native means that you can also use it to develop in the Cloud with tools like [Gitpod ](https://gitpod.io/) and [Github Code Spaces](https://github.com/features/codespaces) etc. In this setup you don't run anything locally anymore, everything runs on a remote server which you access through a browser.

**Reply by alexandreelise** [reply]
In the joomlatools-server definition it says "local/remote development environment" so just to be clear it means it's only to "develop" as a software engineer or a web developer would code but directly on a remote server with docker installed as only requirement. It is not meant to be used as a production server in the cloud. Did I understood correctly Johan?

**Reply by alexandreelise** [reply]
As I understand technically we could use it for production but it's not a good pratice to have dev tools in prod am I correct?

**Reply by johanjanssens** [reply]
Yes, thats correct. Remote means you can run it in the cloud as a development environment, using Gitpod for example. In that setup it doesn't run on your 'local' computer but on a cloud server. 

You can run it in production, nothing prevents you from doing that, but it's not optimised or secured for that kind of scenario.

**Reply by alexandreelise** [reply]
That's the answer I was looking for. Thanks for more insights on your awesome tool Johan. In fact I was lately trying to search for such a tool. There I found it. After some tweaks here and there. Hopefully it will work for my use case. Thanks to Timble Team and Johan Janssens. Strive for the best work possible. Have a delightful day. Take care folks!
